src/Engine/Conditions.h

```diff
@@ -31,6 +32,17 @@ enum class Condition : uint32_t {
 };
 using enum Condition;
 
+class EquipemntPair {
```

Comment:
**Equipment** I believe X)

Reply:
Yeah well, I was just moving the code, you know X)

Reply:
That feel when you rename it in code, is about to commit, and see it's become **Equipemnt**